### PR TITLE
added global git command args support

### PR DESCRIPTION
**Pull request by lunny**
Since go-gitea/gitea#122 depends on this, I sent it.

### CODE REVIEW

File: command.go [hunk 1]
```diff
@@ -13,6 +13,11 @@ import (
 	"time"
 )
 
+var (
+	// GlobalCommandArgs global command args for external package setting
```
Comment (strk) [inline command.go]: // GlobalCommandArgs are arguments appended to the end of every NewCommand

**Comment by strk** [comment]
LGTM

**Comment by 0xbaadf00d** [comment]
LGTM

**Comment by lunny** [comment]
Why `continuous-integration/drone/push` is `waiting for s...`

**Comment by tboerger** [comment]
Push should only wait for pushes to go-gitea/git

**Comment by tboerger** [comment]
LGTM

**Comment by lunny** [comment]
... So let me remove it from PR required.

**Pull request merged**